hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/server/http/ProfileServlet.java

```diff
@@ -500,7 +515,8 @@ public static Event fromInternalName(final String name) {
     COLLAPSED,
     SVG,
     TREE,
-    JFR
+    JFR,
+    HTML
```

Comment:
Async Profiler 2.0 does not seem to create useful output with `-o html`:

```
$ /opt/profiler/profiler.sh -d 10 -o html -f /tmp/prof-output/async-prof-pid-7-cpu-1.html 7
Profiling for 10 seconds
Done
$ cat /tmp/prof-output/async-prof-pid-7-cpu-1.html
Stopped profiling after 10 seconds. No dump options specified
```

We need `-o flamegraph` instead:

```
$ /opt/profiler/profiler.sh -d 10 -o flamegraph -f /tmp/prof-output/async-prof-pid-7-cpu-2.html 7
Profiling for 10 seconds
Done
$ head -3 /tmp/prof-output/async-prof-pid-7-cpu-2.html
<?xml version="1.0" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" width="1200" height="518" onload="init(evt)" viewBox="0 0 1200 518" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Reply:
Thanks for tips. :")